core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/recipe/CcmKeyDeregisterSuccess.java

```diff
@@ -0,0 +1,10 @@
+package com.sequenceiq.cloudbreak.reactor.api.event.recipe;
+
+import com.sequenceiq.cloudbreak.reactor.api.event.StackEvent;
+
+public class CcmKeyDeregisterSuccess extends StackEvent {
```

Comment:
This should probably not be called "Success" because the handler triggers the event even if deregistration fails. "Finished" makes sense, as used in the FreeIPA service.

Reply:
I struggled with the diamond problem in this code, and had to choose between making it consistent with the other classes in the same package (e.g. ClusterProxyDeregisterSuccess), or with the parallel classes in FreeIPA. Given that the code structure matches the other classes in the package, I will leave this as is.